src/shared/components/common/tabs.tsx

```diff
@@ -10,44 +11,42 @@ interface TabsProps {
   tabs: TabItem[];
 }
 
-interface TabsState {
-  currentTab: string;
-}
-
-function handleSwitchTab({ ctx, tab }: { ctx: Tabs; tab: string }) {
-  ctx.setState({ currentTab: tab });
-}
-
-export default class Tabs extends Component<TabsProps, TabsState> {
+export default class Tabs extends Component<TabsProps> {
   constructor(props: TabsProps, context) {
     super(props, context);
-
-    this.state = {
-      currentTab: props.tabs.length > 0 ? props.tabs[0].key : "",
-    };
   }
 
   render() {
     return (
       <div>
-        <ul className="nav nav-tabs mb-2">
-          {this.props.tabs.map(({ key, label }) => (
+        <ul className="nav nav-tabs mb-2" role="tablist">
+          {this.props.tabs.map(({ key, label }, index) => (
             <li key={key} className="nav-item">
               <button
                 type="button"
-                className={`nav-link btn${
-                  this.state?.currentTab === key ? " active" : ""
-                }`}
-                onClick={linkEvent({ ctx: this, tab: key }, handleSwitchTab)}
+                data-bs-toggle="tab"
+                data-bs-target={`#${key}-tab-pane`}
```

Comment:
Not going to work because we don't import bootstrap js.

Reply:
I'd prefer that we don't import that, and instead use the existing inferno handlers for changing tabs. 

I'm all for using the bootstrap styling, but would prefer using our own javascript for event handling.

Reply:
Makes sense, thanks! Would still like to get that a11y markup in there so I’ll trim down this PR.

Reply:
Also in that case I made a recent change to the collapsing sidebar cards using Bootstrap JS that should be reverted then 😬

Reply:
Thanks. Ya I agree that its pretty, but its a whole nother javascript dep to bring in. I think inferno has some animations now but its not worth it IMO.

Reply:
Yeah definitely understandable. I've removed the Bootstrap JS dependency but kept the a11y markup.